src/renderer/index.js

```diff
@@ -1067,158 +1067,6 @@ var Renderer = Base.extend('Renderer', {
         return config.minWidth;
     },
 
-    /**
-     * @summary Render a single cell.
-     * @param {CanvasRenderingContext2D} gc
-     * @param {CellEvent} cellEvent
-     * @param {string} [prefillColor] If omitted, this is a partial renderer; all other renderers must provide this.
-     * @returns {number} Preferred width of renndered cell.
-     * @private
-     * @memberOf Renderer
-     */
-    _paintCell: function(gc, cellEvent, prefillColor) {
```

Comment:
see the diff from here
https://github.com/EclipseTrading/hypergrid/pull/38/files